lib/src/utils/widgets/linked_in_buttons.dart

```diff
@@ -11,45 +11,45 @@ class LinkedInButtonStandardWidget extends StatelessWidget {
     required this.onTap,
     this.iconHeight = 30,
     this.iconWeight = 30,
-    this.iconAssetPath = 'assets/linked_in_logo.png',
-    this.buttonText = 'Sign in with LinkedIn',
-    this.buttonColor = Colors.white,
-    this.textPadding = const EdgeInsets.all(4),
+    this.buttonText = '',
+    this.textStyle,
+    this.textPadding = const EdgeInsets.symmetric(horizontal: 4),
+    this.iconPath,
+    this.backgroundColor,
   });
 
   final Function onTap;
   final double iconHeight, iconWeight;
-  final String iconAssetPath;
   final String buttonText;
-  final Color buttonColor;
   final EdgeInsets textPadding;
+  final TextStyle? textStyle;
+  final String? iconPath;
+  final Color? backgroundColor;
 
   @override
   Widget build(BuildContext context) => Material(
         child: InkWell(
           onTap: onTap as void Function()?,
           child: Container(
-            color: Colors.blue,
+            color: backgroundColor,
```

Comment:
`backgroundColor` should have default color set to blue, as it might be breaking change